main.tf

```diff
@@ -384,12 +382,11 @@ data "aws_iam_policy_document" "ecs_task_access_secrets" {
   statement {
     effect = "Allow"
 
-    resources = [
-      "arn:${var.aws_ssm_path}:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${var.webhook_ssm_parameter_name}",
-      "arn:${var.aws_ssm_path}:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${var.atlantis_github_user_token_ssm_parameter_name}",
-      "arn:${var.aws_ssm_path}:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${var.atlantis_gitlab_user_token_ssm_parameter_name}",
-      "arn:${var.aws_ssm_path}:ssm:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:parameter${var.atlantis_bitbucket_user_token_ssm_parameter_name}",
-    ]
+    resources = coalescelist(
```

Comment:
This will find the first non-empty list.

Maybe unrelated question... Is it possible to make this module work for multiple types of VCS at the same time with little or no changes?

Reply:
fixed this with bt replacing `coalescelist` with `flatten(list())` this works fine for create. but it didnt seem to work right with update. it only saw diff for update the second run

Reply:
to the other question, i don't think so... there are things like `secret_name_key` that are specific to the input and will default to one of the first parameters that are listed